src/views/LimitOrders/hooks/useGelatoLimitOrdersHistory.ts

```diff
@@ -17,12 +17,21 @@ function syncOrderToLocalStorage({ chainId, account, orders }) {
   const ordersLS = getLSOrders(chainId, account)
 
   orders.forEach((order: Order) => {
```

Comment:
It seems we could combiine 2 logics by simply updating the latest orders if latest orders and cache orders are different.
 
 ```
 const ordersLS = getLSOrders(chainId, account)
 
 if (stringify(ordersLS) !== stringify(orders) {
   saveOrders(chainId, account, orders)
 }
 ```

Reply:
I am a bit reluctant to do this because stringify can become a heavy operation if account has too many orders

Reply:
Yeah. Agreed. It's just a simple way to do it. I will think about another optimal diff algorithm and override the current orders with the latest order.

Currently, we loop and store order one by one in local storage.

Reply:
I added saveOrders and removeOrders methods